doc/plugins/authz-keycloak.md

```diff
@@ -17,7 +17,7 @@
 #
 -->
 
-[中文](../zh-cn/plugins/authz-keycloak.md)
+# [中文](../zh-cn/plugins/authz-keycloak.md)
```

Comment:
I think it is much better for the content, not the title.

Reply:
you are right, it has been corrected.